### PR TITLE
Add improvements to The Pit, Fix missing light layer for guns

### PR DESCRIPTION
**Pull request by Fueredoriku**
- Add new walls that obscure players from enemy spawnpoints from their own
- Add new props and decals to create more memorable areas for navigation
- Add triplanar projection toon material and subgraph
- Fix terrain and lighting in the pit so the map looks more like crater town
- Fix correct lightlayers for multiple augments and objects for correct decals
- Fix AIs getting stuck at the bottom of the pit by adding an AI only deathbox
- Fix colliders of chips and ammo boxed being slightly too small

### CODE REVIEW

File: Assets/Prefabs/Interactables/CollectableAmmo.prefab [hunk 1]
```diff
@@ -182,8 +183,8 @@ BoxCollider:
   m_ProvidesContacts: 0
   m_Enabled: 1
   serializedVersion: 3
-  m_Size: {x: 0.7, y: 0.95608616, z: 0.7}
-  m_Center: {x: 0, y: 0.7280432, z: 0}
+  m_Size: {x: 2, y: 2, z: 2}
+  m_Center: {x: 0, y: 1, z: 0}
```
Comment (toberge) [inline Assets/Prefabs/Interactables/CollectableAmmo.prefab:187]: hva skjedd her? du scala opp collideren? isåfall støtter jeg det veldig :+1:

File: Assets/Prefabs/Interactables/CollectableChip.prefab [hunk 2]
```diff
@@ -180,7 +180,7 @@ BoxCollider:
   m_ProvidesContacts: 0
   m_Enabled: 1
   serializedVersion: 3
-  m_Size: {x: 1, y: 1, z: 1}
+  m_Size: {x: 2, y: 2, z: 2}
```
Comment (toberge) [inline Assets/Prefabs/Interactables/CollectableChip.prefab:183]: samme her right?

File: Assets/Scripts/Interactables/DestroyOnDeath.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DestroyOnDeath : MonoBehaviour
```
Comment (toberge) [inline Assets/Scripts/Interactables/DestroyOnDeath.cs:6]: hvor er dette brukt hen?

**Review by toberge — APPROVED** [review]
overall very good, but we should playtest and possibly alter the wall obstacles

File: Assets/Scripts/Interactables/DestroyOnDeath.cs (hunk 3 above)
Comment (Fueredoriku) [reply Assets/Scripts/Interactables/DestroyOnDeath.cs:6]: CBT-skiltene, destroyer hingejointen

File: Assets/Prefabs/Interactables/CollectableAmmo.prefab (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Prefabs/Interactables/CollectableAmmo.prefab:187]: Jepp! Har fått så mye feedback om det, så tenkte å gjøre det mens jeg husket, men så glemte jeg å skrive det ned <.<

File: Assets/Prefabs/Interactables/CollectableChip.prefab (hunk 2 above)
Comment (Fueredoriku) [reply Assets/Prefabs/Interactables/CollectableChip.prefab:183]: Jupp

**Pull request merged**